tests/test_post_to_studio.py

```diff
@@ -169,14 +180,15 @@ def test_post_to_studio_dvc_studio_config(
     monkeypatch.setenv(DVC_EXP_BASELINE_REV, "f" * 40)
     monkeypatch.setenv(DVC_EXP_NAME, "bar")
     monkeypatch.setenv(DVC_ROOT, tmp_dir)
+    monkeypatch.delenv(DVC_STUDIO_TOKEN)
 
     mocked_dvc_repo.config = {"studio": {"token": "token"}}
 
     with Live() as live:
         live.log_metric("foo", 1)
         live.next_step()
 
-    assert mocked_post.call_count == 2
+    assert mocked_post.call_args.kwargs["headers"]["Authorization"] == "token token"
```

Comment:
This test wasn't actually working as intended since the env var was overriding the dvc config. Now it actually checks the token value to make sure it uses the dvc config value.